src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -515,10 +493,41 @@ public function processElementforManifestURL($delta, array $jsondata, FieldItemI
    * Returns whether the entity is indexed in Solr and processed by OCR.
    *
    * @return bool
-   *  TRUE if the entity is found processed, FALSE otherwise
+   *   TRUE if the entity is found processed, FALSE otherwise
    */
   protected function searchEnabled(FieldItemInterface $item): bool {
     return $this->strawberryFieldUtility->getCountByProcessorInSolr($item->getEntity(), 'ocr') > 0;
   }
 
+  /**
+   * Gets the file from the json data mediaitem.
+   *
+   * @param array $mediaitem
+   *   Array of mediaitem data.
+   * @param string $type
+   *   Relevant type for the formatter.
+   *
+   * @return \Drupal\file\FileInterface|bool
+   *   File if found, FALSE otherwise,
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  protected function getFile(array $mediaitem, string $type) {
```

Comment:
I think this needs to use the field_file_drop, opinions? @DiegoPino

Reply:
Hi, `field_file_drop` is very useful but makes picking a Document v/s an Image v/s a 3D Model too complex (means checking the mime types and deciding there after iterating over each one. Its better used when we do not care what type of file we are dealing with but more like "files" in general.

Reply:
I know nothing! 👯